### PR TITLE
update medalla guide: advanced options (WIP)

### PR DESCRIPTION
**Pull request by sachayves**
add subsections on makefile, node parameters, logs, and grafana. minor fixes and edits

### CODE REVIEW

File: docs/the_nimbus_book/src/medalla.md [hunk 1]
```diff
@@ -61,21 +61,8 @@ INF 2020-08-03 16:24:17.951+02:00 Local validator attached                   tid
...
->```
->make NODE_PARAMS="--graffiti='<YOUR_GRAFFITI>'" medalla
->```
-
-> If you are running another client, you may need to use a different port. To change port, run:
->```
->make BASE_PORT=<YOUR_PORT_NUMBER> medalla
->```
-
-</details>
```
Comment (mratsim) [inline docs/the_nimbus_book/src/medalla.md:78]: I think the graffiti is important due to the POAP that requires a specific graffiti

File: docs/the_nimbus_book/src/medalla.md [hunk 2]
```diff
@@ -104,6 +91,29 @@ Once the update is complete, run `make medalla` to reconnect to the network.
...
+
+```
+make NIMFLAGS="-d:insecure" medalla
+```
+
+You can then browse the metrics by connecting to:
+
+http://localhost:8008/metrics
+
+Make sure this port is protected as the http server used is not considered secure (it should not be used by untrusted peers).
```
Comment (mratsim) [inline docs/the_nimbus_book/src/medalla.md:116]: We also need to explain or link to starting our own Grafana instance (it's in the README).

File: docs/the_nimbus_book/src/medalla.md (hunk 1 above)
Comment (sachayves) [reply docs/the_nimbus_book/src/medalla.md:78]: ah, so graffiti is still there (lines 67-69). the explanation was duplicated by accident (once as part of a dropdown, and once directly inline). i've removed the dropdown.

File: docs/the_nimbus_book/src/medalla.md (hunk 2 above)
Comment (sachayves) [reply docs/the_nimbus_book/src/medalla.md:116]: sweet. working on getting it up and running

**Comment by sachayves** [comment]
@mratsim experiencing a little trouble getting grafana up and running. going to merge as is, and open a new PR for grafana + prometheus flow.

**Pull request merged**